hdkeychain/extendedkey.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/decred/dcrd/chaincfg"
 	"github.com/decred/dcrd/chaincfg/chainec"
```

Comment:
Something must still be using this or the code would fail to compile.  Should remove all references.